OpenRA.Mods.Common/Traits/Buildings/RepairableBuilding.cs

```diff
@@ -60,7 +60,7 @@ public class RepairableBuilding : ConditionalTrait<RepairableBuildingInfo>, ITic
 		public RepairableBuilding(Actor self, RepairableBuildingInfo info)
 			: base(info)
 		{
-			health = self.Trait<Health>();
+			health = self.Trait<IHealth>();
```

Comment:
Latent trait-ordering bug here; we should grab this in the created hook instead.

Reply:
Nevermind, I missed the Requires.